### PR TITLE
WIP Upgrade demo to new cerebral 2 features

### PR DESCRIPTION
**Pull request by saitodisse**
- [x] refactor(demos): replaces `input` with `props`
- [x] fix login/signin

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "react": "^15.4.2",
     "react-addons-test-utils": "^15.4.2",
     "react-dom": "^15.4.2",
-    "react-scripts": "0.8.5",
+    "react-scripts": "0.9.0",
```
Comment (saitodisse) [inline package.json]: This enables source maps as discussed here: https://github.com/facebookincubator/create-react-app/issues/343

image

**Comment by Guria** [comment]
is it ready to merge?

**Comment by gaspard** [comment]
@Guria I have to do some more work on it. Will notify when ready...

**Comment by gaspard** [comment]
@saitodisse the login issue is actually a firebase-provider bug.

https://github.com/cerebral/cerebral/pull/684.

**Comment by gaspard** [comment]
@Guria demo seems to work fine now. I think it can be pulled back in.

**Pull request merged**